static/js/chart/draw.ts

```diff
@@ -159,7 +160,8 @@ function showTooltip(
 function getTooltipContent(
   dataGroupsDict: { [place: string]: DataGroup[] },
   highlightedTime: number,
-  unit?: string
+  unit?: string,
+  statsVarInfo?: { [key: string]: StatVarInfo }
```

Comment:
nit: rename to statVarInfo for (new) consistency?

Reply:
done